airbrush.lua

```diff
@@ -71,12 +71,15 @@ function unifieddyes.on_airbrush(itemstack, player, pointed_thing)
 
 	local idx, hue = unifieddyes.getpaletteidx(painting_with, palette)
 	local inv = player:get_inventory()
-	if (not creative or not creative.is_enabled_for(player_name)) and not inv:contains_item("main", painting_with) then
+	if (not minetest.is_creative_enabled(player_name)) and not inv:contains_item("main", painting_with) then
```

Comment:
Need to bump `min_minetest_version` to 5.3.0 for this.

Reply:
https://github.com/mt-mods/unifieddyes/pull/5/commits/608c80662a006927878fce4249ff3c71157347b9